okhttp-android-support/src/test/java/okhttp3/internal/huc/ResponseCacheTest.java

```diff
@@ -269,6 +270,8 @@ private void testResponseCaching(TransferKind transferKind) throws IOException {
   }
 
   @Test public void secureResponseCaching() throws IOException {
+    assumeFalse(getPlatform().equals("jdk9"));
```

Comment:
`"jdk9".equals(getPlatform())` would not throw a NPE when `getPlatform()` returns null